playbooks/configure-apt-sources.yml

```diff
@@ -171,7 +171,16 @@
       when:
         - (apt_artifact_found | bool) and (apt_sources_configure | changed or apt_sources_configure_rpco | changed)
         - apt_artifact_enabled | bool
-
+  roles:
+    - role: "apt_package_pinning"
+      apt_package_pinning_file_name: "rpc_openstack_pinned_packages.pref"
+      apt_pinned_packages:
+        - package: "*"
+          release: "{{ rpc_release }}-{{ ansible_distribution_release }}"
+          priority: "998"
+        - package: "*"
+          release: "{{ ansible_distribution_release }}"
```

Comment:
Why is this a higher priority? I'm a little confused about the intention here? I thought that the artifacts should be highest priority?

We could also set this to priority 1001 which would cause forcible downgrades if something else is installed. This might help us with the day 1 deployment issues?

Reply:
Yup I've these backwards and I'm not sure if I need the second entry at all.

As for the priority, I considered doing 1001 but the general consensus I got from folks is dowgraded packages in any form is a bad idea and I tend to agree.